src/main/java/com/minecolonies/coremod/client/gui/WindowCitizen.java

```diff
@@ -544,6 +575,46 @@ private void createSkillContent()
           LanguageHandler.format("com.minecolonies.coremod.gui.citizen.skills.dexterity", citizen.getDexterity()));
     }
 
+    private ImmutableList<RequestWrapper> getOpenRequestTreeOfCitizen()
+    {
+        final ColonyView colonyView = ColonyManager.getClosestColonyView(FMLClientHandler.instance().getWorldClient(),
+          (citizen.getWorkBuilding() != null) ? citizen.getWorkBuilding() : citizen
+                                                                              .getHomeBuilding());
+        if (colonyView == null)
+        {
+            return ImmutableList.of();
+        }
+
+        final List<RequestWrapper> treeElements = new ArrayList<>();
+
+        getOpenRequestsOfCitizen().stream().forEach(r -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1602)